docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes-provider.asciidoc

```diff
@@ -331,6 +331,81 @@ The policy generated by this configuration looks like:
         target: orchestrator
 ----
 
+[discrete]
+== Autodiscover targeted Pods using hints (standalone mode)
+
+Standalone Elastic Agent supports autodiscover based on hints from the provider.
+The hints system looks for hints in Kubernetes Pod annotations which have the prefix `co.elastic.hints`.
+As soon as the container starts, Elastic Agent will check if it contains any hints and launch the proper config for it.
+Hints tell Elastic Agent how to monitor the given container by using the proper integration.
+This is the full list of supported hints:
+
+[float]
+===== `co.elastic.hints/package`
+
+The package to use for monitoring.
+
+[float]
+===== `co.elastic.hints/data_stream`
+
+(optional) The list of data_streams to enable. If not specified the default data_streams will be enabled using the
+default values as specified in the respective package/integration.
+
+If data_streams are specified, then hints can be defined per data_stream like `co.elastic.hints/key.period: 10m`,
+otherwise the data_stream will use the top level hints (`co.elastic.hints/period: 10m`) in its configuration.
+
+[float]
+===== `co.elastic.hints/host`
+
+The host to use for metrics retrieval.
+
+[float]
+===== `co.elastic.hints/metrics_path`
+
+The path to retrieve the metrics from (/metrics by default).
+
+[float]
+===== `co.elastic.hints/period`
+
+The time interval for metrics retrieval, ie: 10s
+
+[float]
+===== `co.elastic.hints/timeout`
+
+Metrics retrieval timeout, default: 3s
+
+[float]
+===== `co.elastic.hints/username`
+
+The username to use for authentication
+
+[float]
+===== `co.elastic.hints/password`
+
+The password to use for authentication. It is recommended to retrieve this sensitive information from an ENV variable
+and avoid placing passwords in plain text.
+
+[float]
+===== `co.elastic.hints/stream`
+
+The stream to use for logs collection, ie stdout/stderr.
+
+In any case if the specified package has no logs support the generic `container_logs` input will be used as a
```

Comment:
should it be '`container_logs` input' or '`container_logs` datastream of kubernetes package'?

Reply:
It's the a generic container logs input section, see https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/elastic-agent-standalone/templates.d/prometheus.yml#L70-L90 as an example.